common/pkcs11.h

```diff
@@ -982,7 +982,6 @@ struct ck_aes_ctr_params {
 struct ck_gcm_params {
   unsigned char *iv_ptr;
   unsigned long iv_len;
-  unsigned long iv_bits;
```

Comment:
Seems like there was a confusion in the PKCS#11 standard, where the doc says nothing about this field, while the official C [header](https://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/errata01/os/include/pkcs11-v2.40/pkcs11t.h) has this (see the [comment](https://hg.mozilla.org/projects/nss/file/tip/lib/util/pkcs11t.h#l1877) in NSS for the rationale).

Given the definition is copied into SoftHSMv2 and used already, I'd suggest not touching it.

Reply:
OK, I guess I then also have to adapt Opencryptoki to maybe support both structure versions.....

Reply:
changed, see updated commit [Add support for CKM_AES_GCM](https://github.com/p11-glue/p11-kit/pull/415/commits/949daa1289ff484cad134edc20c98b12ee2395bc).